dart_challenge/lib/game_3.dart

```diff
@@ -0,0 +1,46 @@
+import 'dart:io';
+import 'dart:math';
+
+void game3() {
+  int cows = 0, bulls = 0, attempts = 0, oldCows = 0;
+  Random random = new Random();
+  List<String> listNumber = [(random.nextInt(9) + 1).toString()];
+  for (var i = 0; i < 3; i++) {
```

Comment:
Thuật toán này đúng nhưng vẫn hơi dài, nếu nghĩ được cách làm với 1 dòng random thì fix ko thì bỏ qua cũng được nha